apps/ledger-live-desktop/src/renderer/components/OperationsList/AmountCell.jsx

```diff
@@ -68,7 +68,13 @@ class AmountCell extends PureComponent<Props> {
                   showCode
                   fontSize={4}
                   alwaysShowSign
-                  color={amount.isNegative() ? "palette.text.shade80" : undefined}
+                  color={
```

Comment:
Would it be more concise if
```
operation.type === "OUT"? "palette.text.shade80" : isConfirmed ? undefined : colors.warning
```
?

Reply:
I understand what you want to do but i think it's better to separate the case "not confirmed and incoming", discussed on slacked